packages/scalars/src/scalars/index.ts

```diff
@@ -0,0 +1,20 @@
+// namespace imports -- DO NOT REMOVE OR EDIT THIS COMMENT
```

Comment:
Let's keep it like this but this could also be generated when bundling.

Reply:
I'm using this comments as reference for the generator, so it can update the index file properly